golos.vesting/golos.vesting.cpp

```diff
@@ -168,12 +168,13 @@ void vesting::cancel_convert_vesting(account_name sender, asset type) {
     table.erase(record);
 }
 
-void vesting::delegate_vesting(account_name sender, account_name recipient, asset quantity, uint16_t percentage_deductions) {
+void vesting::delegate_vesting(account_name sender, account_name recipient, asset quantity, uint16_t interest_rate, uint8_t payout_strategy) {
     require_auth(sender);
     eosio_assert(sender != recipient, "You can not delegate to yourself");
 
-    eosio_assert(MIN_AMOUNT_DELEGATION_VESTING <= quantity.amount, "Insufficient funds for delegation");
-    eosio_assert(percentage_deductions <= MAX_PERSENT_DELEGATION, "Exceeded the percentage of delegated vesting");
+    eosio_assert(quantity.amount >= 0, "the number of tokens should not be less than 0");
+    eosio_assert(quantity.amount >= MIN_AMOUNT_DELEGATION_VESTING, "Insufficient funds for delegation");
+    eosio_assert(interest_rate <= MAX_PERSENT_DELEGATION, "Exceeded the percentage of delegated vesting");
```

Comment:
add check of `MIN_DELEGATION`. current received+quantity must be >= `MIN_DELEGATION` (`MIN_DELEGATION` will be dynamic)